packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -242,12 +271,12 @@ class Radio extends React.Component<Props> {
     };
 
     render(): React.Node {
-        let choices = this.props.choices;
-        let choiceStates;
+        const {choices} = this.props;
+        let choiceStates: $ReadOnlyArray<ChoiceState>;
         if (this.props.static) {
-            choiceStates = _.map(choices, (val) => ({
-                selected: val.correct,
-                crossedOut: val.crossedOut,
+            choiceStates = choices.map((choice) => ({
+                selected: !!choice.correct,
+                crossedOut: false,
```

Comment:
This seems to be changing logic, is that intentional?

Reply:
I suggested this change.  I don't see how `crossedOut` could be a member of `choice` so it seems like this should always be false and the tests are passing. 🤷‍♂️

Reply:
Is this what you would like to see a test for, @handeyeco?

Reply:
I see, because this is initializing the choiceStates? Or maybe static doesn't have the concept of interactivity? So it was likely always undefined and thus falsey. Okay, NAN.